types/tipset_key.go

```diff
@@ -128,6 +131,17 @@ func (s TipSetKey) String() string {
 	return out + " }"
 }
 
+// Serialize returns a serialize string of TipSetKey use gob and base64.
+func (s TipSetKey) Serialize() string {
+	b := bytes.Buffer{}
+	e := gob.NewEncoder(&b)
+	err := e.Encode(s)
+	if err != nil {
+		return ""
```

Comment:
in the off chance that we leave this method on `TipSetKey`, we should **not** be swallowing this error.